.env.example

```diff
@@ -2,4 +2,5 @@
 
 TENANT_ID=<azure_tenant_id>
 SUBSCRIPTION_ID=<azure_subscription id>
-INFRA_NAMES="basic cluster"
\ No newline at end of file
+INFRA_NAMES="basic cluster"
+SERVICE_PRINCIPAL_APP_OBJ_ID="<azure_app_registration_object_id>"
```

Comment:
we should have some details on how to generate this in /docs

Reply:
or maybe even a script that generates it for us

Reply:
good point i'll add some

Reply:
also we only need one of these per tenant, which will realistically map to one per cloud

Reply:
added